src/endpoints/get-diffs.js

```diff
@@ -17,33 +18,40 @@ module.exports = cors(async (req, res) => {
     return error(res, 400, `Invalid ?since=${query(req).since}`)
   }
 
-  const sessionStmt = db.prepare('SELECT * FROM latest_session_state WHERE short_id = ? LIMIT 1');
-  const session = sessionStmt.get(sessionId);
+  const sessionStmt = db.prepare(
+    "SELECT * FROM latest_session_state WHERE short_id = ? LIMIT 1"
+  )
+  const session = sessionStmt.get(sessionId)
 
   if (!session) return error(res, 404, "Session Not Found")
 
-  const samplesQueryResults = db.prepare(`SELECT  *
-                           FROM latest_sample_state
-                           WHERE session_short_id = ?`).all(session.short_id);
+  const samplesQueryResults = db
+    .prepare(
+      ` SELECT  *
+                                           FROM sample_state
+                                           WHERE session_short_id = ?`
+    )
+    .all(session.short_id)
 
   const samples = []
-  samplesQueryResults.forEach(sample => {
-    const content = JSON.parse(sample.content)
-    const annotation = JSON.parse(sample.annotation)
-    samples.push(Object.assign({}, content, {annotation}))
+  samplesQueryResults.forEach((sample) => {
+    const sessionUDT = getSessionUDT(sample)
```

Comment:
hmm this confuses me. why would you call `getSessionUDT` with a sample...

Reply:
renamed to `getSampleObject`